src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,151 @@
+package Duke;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+public class Duke {
+    public static final String Line = "    ____________________________________________________________";
+    private static ArrayList<Task> tasks = new ArrayList<>();
+    private static int taskSum = 0;
+    public static void printList() {
+        Task now;
+        System.out.println(Line);
+        System.out.println("     Here are the tasks in your list:");
+        for(int i=0; i<taskSum; i++){
+            now = tasks.get(i);
+            System.out.println("     " + (i+1) + "." + now.toString());
+        }
+        System.out.println(Line);
+    }
+
+    public static void bye() {
+        System.out.println(Line + "\n" + "     Bye. Hope to see you again soon!" + "\n" + Line);
+    }
+
+    public static void greeting() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println(Line);
+        System.out.println("     Hello! I'm Duke");
+        System.out.println("     What can I do for you?");
+        System.out.println(Line);
+    }
+
+    private static void writeToFile(String filePath) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        for(int i=0; i<taskSum; i++){
+            Task now = tasks.get(i);
+            fw.write(String.format("%d ",i+1));
+            fw.write(now.toString());
+            fw.write("\n");
+        }
+        fw.close();
+    }
+
+    public static void main(String[] args) {
+        greeting();
+        String command;
+        Scanner in = new Scanner(System.in);
+        String f = "data/lines.txt";
```

Comment:
If the string is a constant, the variable name must be all uppercase (and underscore to separate the words) instead. A better variable name would also make understanding the code easier.